### PR TITLE
Added kubeConfig stanza to default gke config

### PR DESCRIPTION
**Pull request by guineveresaenger**

### CODE REVIEW

**Comment by davidewatson** [comment]
please test this

**Comment by coffeepac** [comment]
ok to test

**Comment by coffeepac** [comment]
ok to test

File: k2-configs/default-pr-gke.yaml [hunk 1]
```diff
@@ -4,9 +4,14 @@ deployment:
...
       name: k2key
-      providerConfig: 
+      providerConfig:
         serviceAccount: "{{ lookup('env','GCE_SERVICE_ACCOUNT_ID') }}"
         serviceAccountKeyFile: "{{ lookup('env','GCE_SERVICE_ACCOUNT_KEY') }}"
+  kubeConfig:
+    -
+      name: krakenKubeConfig
+      version: 1.5.6
+      hyperkubeLocation: gcr.io/google_containers/hyperkube
```
Comment (davidewatson) [inline k2-configs/default-pr-gke.yaml]: I don't think this line is necessary for GKE.

Comment (davidewatson) [inline k2-configs/default-pr-gke.yaml]: s/this line/hyperkubeLocation/g

Comment (guineveresaenger) [inline k2-configs/default-pr-gke.yaml]: I'm inclined to believe you. I left it in because I don't know what it's for.

**Pull request merged**